backend/internal/controllers/posts_controller.go

```diff
@@ -18,3 +20,23 @@ func GetPosts(ctx *gin.Context) {
 		handleError(ctx, 404, errors.New("not found"))
 	}
 }
+
+func GetPostById(ctx *gin.Context) {
+	idString := ctx.Param("id")
+	id, err := strconv.Atoi(idString)
+	if err != nil {
+		ctx.String(http.StatusBadRequest, "not an integer")
+		return
+	}
+	// ctx.String(http.StatusOK, greetings[id])
+
+	u := usecases.NewGetPostByIdUsecase(ctx)
+	result, err := u.Execute(id)
+	if err != nil {
+		handleError(ctx, 500, err)
```

Comment:
こんな感じで、数字じゃなくて意味もった単語で渡せるよ。多分、、、
```suggestion
		handleError(ctx, http.StatusInternalServerError, err)
```